backend/molgenis-emx2-graphql/src/main/java/org/molgenis/emx2/graphql/GraphqlDatabaseFieldFactory.java

```diff
@@ -44,6 +44,10 @@ public GraphQLFieldDefinition.Builder createMutation(Database database) {
             GraphQLArgument.newArgument().name(GraphqlConstants.NAME).type(Scalars.GraphQLString))
         .argument(
             GraphQLArgument.newArgument().name(Constants.DESCRIPTION).type(Scalars.GraphQLString))
+        .argument(
```

Comment:
thanks a lot for implementing this, because I asked for it. And sorry for the 'settings' question above but I think it helps for ourselves to have clarity on when to use settings and when to add a seperate metadata item.